### DISCUSSION TITLE
Categories not working

### DISCUSSION THREAD

**Discussion by Albert24GG** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#283] [Q&A] [answered]
When I change the categories from the settings.yaml file and I re-run the script, it still searches for the same old categories.
![image](https://user-images.githubusercontent.com/79106176/117469585-936b4700-af5e-11eb-9328-52107ea57b67.png)
![image](https://user-images.githubusercontent.com/79106176/117469612-9b2aeb80-af5e-11eb-8f48-65cf788c28e4.png)

**Comment by cullzie** [comment, marked as answer]
Hey @Albert24GG, I think this is due to the ordering of the checks. We check if we have already enrolled before checking the category hence you're seeing that message output. We could move the category check before the already enrolled check to avoid this output.
 
The categories logic should still be working though. If it isn't please raise a bug on it with the relevant info. Thanks
